pkgs/by-name/ku/kubectl-get-all/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "kubectl-get-all";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "stackitcloud";
+    repo = pname;
```

```suggestion
    repo = "kubectl-get-all";
```
pname like this is an antipattern

---

Thanks for the info